src/ReachSets/DiscretePost/DiscretePost.jl

```diff
@@ -113,3 +113,12 @@ function get_overapproximation_option(op::DiscretePost, n::Int)
         error("received unknown :overapproximation option $oa")
     end
 end
+
+# --- default methods for handling assignments ---
+
+function apply_assignment(𝒫::DiscretePost,
+                          constrained_map::AbstractMap,
```

Comment:
```suggestion
                          constrained_map::Union{LinearMap, ConstrainedLinearMap},
```

Reply:
The idea was that this is the generic method that potentially "just works". For instance, it also works for the `IdentityMap`.